src/app/clusters/window-covering-server/window-covering-server.cpp

```diff
@@ -649,6 +593,21 @@ EmberAfStatus GetMotionLockStatus(chip::EndpointId endpoint)
     return EMBER_ZCL_STATUS_SUCCESS;
 }
 
+void SetDefaultDelegate(EndpointId endpoint, Delegate * delegate)
+{
+    uint16_t ep = emberAfFindClusterServerEndpointIndex(endpoint, Channel::Id);
+
+    // if endpoint is found and is not a dynamic endpoint
+    if (ep != 0xFFFF && ep < EMBER_AF_WINDOW_COVERING_CLUSTER_SERVER_ENDPOINT_COUNT)
```

Comment:
is there a constant name we could define for 0xFFFF? It would be easier to read if it was called something like `kInvalidEndpointId` or similar.